### ISSUE TITLE
Error 'could not store action: Illegal primitive value for property date', after trying to adding reference to existing dataobject

### ISSUE THREAD

**Issue by laura-ham** [weaviate/weaviate#991]
The POST request to http://localhost:8080/v1/actions/22cc3380-ef73-48f8-9e3c-c603fae0f4b0/references/toCompany leads to the error message:
```json
{
    "error": [
        {
            "message": "could not store action: Illegal primitive value for property date, value is \"2019-03-24T14:25:57Z\""
        }
    ]
}
```

The data object where the reference should be added to has a property `date` with the `dataType` `date`, which is filled with the abovementioned date. This is previously added to the Weaviate instance without any problems. Adding a reference to another property of this same action class however suddenly gives an error message about the date.

**Comment by bobvanluijt** [comment]
Still a relevant bug

**Comment by etiennedi** [comment]
Fair point, I'm configuring the stale bot not to close issues with the label `bug`, as bugs don't just magically disappear on their own (unfortunately).

**Comment by etiennedi** [comment]
> Still a relevant bug

@bobvanluijt How did you determine this? Based on the description I would have expected that this was fixed with #1069, but it's hard to validate without building a reproducing example. So if you already have one, please share.

**Comment by bobvanluijt** [comment]
Because it is a relevant feature, didn't test it but assumed it from the fact that it was still open and had no commits.
